File: usr/benchmark1.cs
using System;

public class Benchmark{
  
  public static void Main(string[] args){
 
    //Start timming
    var Timer=System.Diagnostics.Stopwatch.StartNew();

    //Variables
    int i;
    int j;
    int z;

    //Calculation
    z=0;
    i=0;
    while(i<10000){
      j=0;
      while(j<1000){
        z=2*z+j;
        j++;
      }
      i++;
    }

    //End timming
    Timer.Stop();
    Console.WriteLine("CS Benchmark: "+(((float)Timer.ElapsedTicks/(float)TimeSpan.TicksPerSecond).ToString("0.00000")));
    Console.WriteLine(z.ToString());

  }

}

File: usr/benchmark1.py
from timeit import default_timer as timer
start = timer()
z=0
i=0
while i<10000:
  z=0
  j=0
  while j<1000:
    z=2*z+j
    j+=1
  i+=1
end = timer()
print("PY Benchmark: {0:.5f}s".format(end-start))
